serverlessworkflow/antora.yml

```diff
@@ -47,6 +47,7 @@ asciidoc:
     sonataflow_devmode_imagename: quay.io/kiegroup/kogito-swf-devmode-nightly
     sonataflow_builder_imagename: quay.io/kiegroup/kogito-swf-builder-nightly
     sonataflow_devmode_devui_url: /q/dev-ui/org.apache.kie.sonataflow.sonataflow-quarkus-devui/
+    sonataflow_management_console_imagename: quay.io/kiegroup/osl-management-console-image
```

Comment:
@ricardozanini I put this address as a placeholder, do you know the right URL where the image will be available?

Reply:
Yes, we have to ask @rodrigonull how we publish this image

Reply:
@ricardozanini Is that image already being built on kie tools? If so, which package?
If it is a new image we need to create a ticket on Apache Infra jira board for them to add the new dockerhub repository.

Reply:
It's a new image: https://github.com/apache/incubator-kie-tools/tree/main/packages/sonataflow-management-console-image

@fantonangeli, can you open the ticket? See this example: https://issues.apache.org/jira/browse/INFRA-25794.

@rodrigonull, what are the next steps after opening the ticket?

Reply:
After the repo is created on dockerhub we need to update the KIE Tools daily dev job and add the image env vars and the the docker push command: https://github.com/apache/incubator-kie-tools/blob/main/.ci/jenkins/Jenkinsfile.daily-dev-publish

Reply:
@fantonangeli ^

Reply:
@ricardozanini @rodrigonull 
Task to update the jenkin job:
https://github.com/apache/incubator-kie-tools/issues/2679
Feel free to assign yourself if you think would be a better fit.

Reply:
Thank you!!!!

Reply:
@ricardozanini @rodrigonull I created the Jenking CI PR: https://github.com/apache/incubator-kie-tools/pull/2689